app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= link_to user_post_path(post.author, post) do %>
+  <article class="post_view">
+    <div class="post__container">
+      <h3><%= post.title %></h3>
+
+      <% if local_assigns[:show_all_comments] %>
+        <div class="post__counters">
+          Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+        </div>
+      <% end %>
```

Comment:
- [ ] To match the provided **[wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)** and as required for the project, kindly make sure you also render the `post's text` text as right now nothing is displayed.